### ISSUE TITLE
Misleading toggle announcement

### ISSUE THREAD

**Issue by aek5ba** [Workday/canvas-kit#3502]
What's the issue
The 'Calendar' and 'Date Range' buttons are separate buttons. However, both use the attribute aria-pressed, which is misleading as it implies that the button can be "pressed" and then "un-pressed" by activating the same trigger, when in actuality the user must press the adjacent button.

Simplified Code Sample

<div>
    <button type="button" aria-pressed="true" title="Calendar"><span>Calendar</span></button>
    <button type="button" aria-pressed="false" title="Date Range"><span>Date Range</span></button>
</div>

Why it matters
Screen reader users will have certain expectations on how to interact with an element or component based what is announced. If an attribute is added but the matching functionality is missing, the user will not be able to properly interact with the element or component.

Our recommendation

Group the buttons so users will know they are related. On the parent <div>, add role="group".
Give the group of buttons a name to clarify the group's purpose. On the parent <div>, add aria-label="Calendar / Date range view toggle" (or select more appropriate text for the aria-label).
On both buttons, replace the aria-pressed attribute with aria-current. Toggle the values between true when the button is active and false when not active.

Code example

<div role="group" aria-label="Calendar / Date range view toggle">
    <button type="button" aria-current="true" title="Calendar"><span>Calendar</span></button>
    <button type="button" aria-current="false" title="Date Range"><span>Date Range</span></button>
</div>

Type
WCAG 2

Success Criteria
1.3.1 Info and Relationships (A)

Severity
Medium

Item
Calendar / Date Range toggle buttons

Section
Modal - Modal as Overlay

Issue ID
WOR-8554

Attachments
[WOR-CH-1331.png](https://workday.audit.intopia.digital/WOR/WOR-8554/WOR-CH-1331.png),[WOR-CH-1344.png|https://workday.audit.intopia.digital/WOR/WOR-8554/WOR-CH-1344.png]

Page:

Core HCM - Absence - Manage Absence
Core HCM - Absence - Request Absence
